app/javascript/tests/layout/Header.test.js

```diff
@@ -0,0 +1,183 @@
+    expect(primaryNav.find('.analytics-tab').exists()).toBe(false);
+    expect(primaryNav.find('.api-tab').exists()).toBe(false);
+    expect(primaryNav.find('.jobs-tab').exists()).toBe(false);
+    expect(wrapper.find(Navbar).find('.banner-message').exists()).toBe(false);
+  });
+
+  it('Shows "Enroller Dashboard" tab when user. can_see_enroller_dashboard_tab', () => {
+    const wrapper = getWrapper({ current_user: mockUser2 });
+    const primaryNav = wrapper.find(Navbar).find('.primary-nav');
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/patients` })
+        .exists()
+    ).toBe(true);
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/patients` })
+        .text()
+    ).toEqual('Enroller Dashboard');
+  });
+
+  it('Shows "Analytics" tab when user. can_see_analytics_tab', () => {
+    const wrapper = getWrapper({ current_user: mockUser2 });
+    const primaryNav = wrapper.find(Navbar).find('.primary-nav');
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/analytics` })
+        .exists()
+    ).toBe(true);
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/analytics` })
+        .text()
+    ).toEqual('Analytics');
+  });
+
+  it('Shows "Monitoring Dashboards" tabs when user. can_see_monitoring_dashboards_tab', () => {
+    const wrapper = getWrapper({ current_user: mockUser2 });
+    const primaryNav = wrapper.find(Navbar).find('.primary-nav');
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/public_health` })
+        .exists()
+    ).toBe(true);
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/public_health` })
+        .text()
+    ).toEqual('Monitoring Dashboards');
+  });
+
+  it('Shows "API" and "Jobs" when user.  is_usa_admin', () => {
+    const wrapper = getWrapper({ current_user: mockUser2 });
+    const primaryNav = wrapper.find(Navbar).find('.primary-nav');
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/oauth/applications` })
+        .exists()
+    ).toBe(true);
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/oauth/applications` })
+        .text()
+    ).toEqual('API');
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/sidekiq` })
+        .exists()
+    ).toBe(true);
+    expect(
+      primaryNav
+        .find(Nav.Link)
+        .find({ href: `${window.BASE_PATH}/sidekiq` })
+        .text()
+    ).toEqual('Jobs');
+  });
+
+  it('Properly updates the active link based on the activeKey', () => {
+    const wrapper = getWrapper({ current_user: mockUser2 });
+    const keys = ['/patients', '/public_health', '/admin', '/analytics'];
+    keys.forEach(key => {
```

Comment:
```suggestion
    _.shuffle(keys).forEach(key => {
```

Reply:
that way this isn't always done in the same order

Reply:
[3ee11e6](https://github.com/SaraAlert/SaraAlert/pull/1243/commits/3ee11e6f2afdbe8e22fd0568893229c5192ff20d)